src/OpenTelemetry.Instrumentation.Runtime/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 * Updated OpenTelemetry core component version(s) to `1.10.0`.
   ([#2317](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2317))
 
+* Remove legacy checks that only applied for .NET 6.
+  ([#2325](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2325))
```

Comment:
I think that it can be merged with first entry under unreleased. 

```md
  ([#2155](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2155),
  [#2325](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/2325))
```

Reply:
Updated.